### ISSUE TITLE
[rancher-dns] How does the DNS handles the relaunch of a dead pod

### ISSUE THREAD

**Issue by 7rouz** [rancher/rancher#5413]
**Rancher Version:**  v1.1.0

**Docker Version:**  1.11.2

**OS and where are the hosts located :** cloud

**Setup Details:** : HA rancher, internal DB

**Environment Type:**  Kubernetes (v1.3)

**Steps to Reproduce:** N/A

**Results:**  N/A

**Expected:**

I am interested in the Rancher with Kubernetes because I have hit some limitations with SkyDNS of Kube. I was wondering how does rancher-dns handle pods' relaunching.
In other words, imagine a pod that is linked to a service fell down and a new pod was created to replace it, how will rancher-dns handle this ?

Thank you in advance.

**Comment by deniseschannon** [comment]
In Rancher on Kubernetes, for every service/pod created in Kubernetes, we create a service and a container (aka pod). Whenever there are changes made in kubernetes pods, we reflect the changes in Rancher and the service's DNS is updated to reflect the pod/container is no longer available or a new one was created.

Let me know if you have further questions.

**Comment by 7rouz** [comment]
Thank you !!
